### PR TITLE
Component: Dropdown

### PR DESCRIPTION
**Pull request by lizlux**
This PR sets up the style guide's first basic web component. It's not fully configurable yet, but it does work. Issue https://github.com/Wikia/style-guide/issues/44 was created so someone can come back to this code to polish, refine, and apply it in other places.

### CODE REVIEW

File: dist/scripts/components/dropdown.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+	proto.name = 'Dropdown Icon';
+
+	proto.createdCallback = function() {
+		// import the template and retrieve a document fragment from it
+		var content = document.querySelector('#dropdownIconImport').import;
+		var template = content.querySelector('#dropdownIconTemplate');
+		var clone = document.importNode(template.content, true);
+
+		// import the list items
+		var list = clone.querySelector('.f-dropdown');
```
Comment (kenkouot) [inline dist/scripts/components/dropdown.js:20]: `.f-dropdown` was the original Foundation class. Can you change it here and in the SCSS to something more appropriate? Maybe `dropdown-menu`

Comment (lizlux) [inline dist/scripts/components/dropdown.js:20]: Added a note to handle this with https://github.com/Wikia/style-guide/issues/44

**Comment by lizlux** [comment]
@bkoval

**Comment by lizlux** [comment]
Closing in favor of https://github.com/Wikia/style-guide/pull/45

**Pull request closed without merging**